src/Controls/src/Core/VisualElement.cs

```diff
@@ -23,9 +23,11 @@ public partial class VisualElement : NavigableElement, IAnimatable, IVisualEleme
 		/// <include file="../../docs/Microsoft.Maui.Controls/VisualElement.xml" path="//Member[@MemberName='InputTransparentProperty']/Docs/*" />
 		public static readonly BindableProperty InputTransparentProperty = BindableProperty.Create("InputTransparent", typeof(bool), typeof(VisualElement), default(bool));
 
+		internal static readonly BindableProperty UnCoercedIsEnabledProperty = BindableProperty.Create("UnCoercedIsEnabled", typeof(bool), typeof(VisualElement), true);
```

Comment:
This may also need to be a non-bindable property... It could just be a normal boolean field...